Documentation/network/kubernetes/compatibility-table.rst

```diff
@@ -62,6 +62,8 @@
 +--------------------+----------------+
 | v1.14.0-pre.2      | 1.26.8         |
 +--------------------+----------------+
+| v1.14.0-rc.0       | 1.26.9         |
++--------------------+----------------+
```

Comment:
Something is off here - we're meant to release `v1.14.0-rc1`, with CRD Schema version `1.26.10`.

And even more confusing, a few lines below we already have

> | v1.14.0-rc.0 | 1.26.10 |

Reply:
Yeah, the script only does all release with git tags so far, so it's always one step behind. This is nothing new. Ideally we'd fix it, but it's pretty rare for this to be a point of concern. It's a best-effort info table.

Reply:
I take that "as no concern, let's ship" :).